javascript/javascript2/week1/indexjs2week1.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Spirit Animal</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <script src="script1.js"></script>
+    <div>
+      <form action="namegenerator" method="GET">
+        <div class="mainDiv">
+          <label for="firstName" id="nameLabel"> First Name: </label>
+          <input type="text" name="firstname" id="firstName" required />
+        </div>
+        <div class="btn">
+          <input
```

Comment:
You could use the `<button>` element instead. It is more up to date way of creatng buttons :) 
If you triued that and had an issue with page reloading that is becasue of a default behavior of `<button>` element withing a `<form>`. it will submit ythe for and reload the page wich is almost never what we want :) 
You can prevent this in couple of ways:
1. use `<button type="button">` and listen to `click` event 
2. alternatively you cal listen to `submit` event on the form and in the event handler (this is how we call the funcion that is called when a certain event is firen and that we pass to `addEventLister` as a second argument) you need to call `prevenDefault()` method on the event itself. many events have default behaviors, so it is good to know how to prevent it from happening.